hw/dv/tools/dvsim/vcs.hjson

```diff
@@ -239,7 +240,9 @@
                    "-cm {cov_metrics}",
                    // Set the coverage hierarchy
                    "{vcs_cov_cfg_file}",
-                   // Set the assert coverage hierarchy
+                   // Set all coverage hierarchy(including assertion coverage) in {vcs_cov_cfg_file}
+                   "-cm_common_hier",
```

Comment:
Will `-cm_common_hier` cause issues if `vcs_cov_cfg_file` is not set?

Reply:
I tried one without vcs_cov_cfg_file, the simulation runs fine with coverage, no errors or warnings. I believe we should be fine.